modules/javafx.graphics/src/main/native-glass/mac/a11y/AccessibleBase.m

```diff
@@ -129,6 +153,30 @@ - (id)accessibilityParent
     return parent;
 }
 
+- (id)accessibilityTitle
+{
+    jobject jresult = NULL;
+    GET_MAIN_JENV;
+    if (env == NULL) return NULL;
+    jresult = (jobject)(*env)->CallLongMethod(env, self->jAccessible,
+                                              jAccessibilityAttributeValue, (jlong)@"AXTitle");
+    GLASS_CHECK_EXCEPTION(env);
+    return variantToID(env, jresult);
+}
```

Comment:
This method is same as `accessibilityLabel`. Should we change implementation to call one method from the other ?

Reply:
Makes sense.